scripts/seed.py

```diff
@@ -0,0 +1,162 @@
+# Command to run : python manage.py shell --settings=settings.dev  < scripts/seed.py
+
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+from allauth.account.models import EmailAddress
+
+from datetime import timedelta
+from faker import Factory
+import random
+
+from challenges.models import Challenge, ChallengePhase, DatasetSplit, Leaderboard, ChallengePhaseSplit
+from hosts.models import ChallengeHostTeam, ChallengeHost
+from participants.models import Participant, ParticipantTeam
+
+
+def create_admin_user():
+    username = "admin"
+    email = "admin@gmail.com"
+    User.objects.create_user(
+        email=email,
+        username=username,
+        password="password",
+        is_staff=True,
+        is_superuser=True,
+    )
+    print "Super user made with Username: admin and Password: password"
```

Comment:
It might be good idea to not hardcode the username and password in the print statements. Please use string formatting here. :)